src/main/java/Duke.java

```diff
@@ -1,10 +1,233 @@
+                break;
+            case "todo":
+                if (commandTokens.length < 2) {
+                    printInsufficientArgumentsMessage();
+                    break;
+                }
+                addTodo(tasks, commandTokens[1]);
+                break;
+            case "deadline":
+                if (commandTokens.length < 3) {
+                    printInsufficientArgumentsMessage();
+                    break;
+                }
+                addDeadline(tasks, commandTokens[1], commandTokens[2]);
+                break;
+            case "event":
+                if (commandTokens.length < 3) {
+                    printInsufficientArgumentsMessage();
+                    break;
+                }
+                addEvent(tasks, commandTokens[1], commandTokens[2]);
+                break;
+            default:
+                /* Unknown command, prompt user to use the "help" command */
+                printFallbackMessage();
+                break;
+            }
+        }
+    }
+
+    private static String[] tokenizeInput(String userCommand) {
+        String[] commandTokens = userCommand.split(" ", 2);
+        /* Handle empty inputs */
+        if (commandTokens.length == 0) {
+            return commandTokens;
+        }
+        /* Make commands case-insensitive */
+        commandTokens[0] = commandTokens[0].toLowerCase();
+        String[] commandParsed = commandTokens;
+        String[] commandArgs;
+        switch (commandTokens[0]) {
+        case "deadline":
+            if (commandTokens.length > 1) {
+                commandArgs = commandTokens[1].split("/by");
+                commandParsed = new String[commandArgs.length + 1];
+                commandParsed[0] = commandTokens[0];
+                for (int i = 0; i < commandArgs.length; i++) {
+                    commandParsed[i + 1] = commandArgs[i];
+                }
+            }
+            break;
+        case "event":
+            if (commandTokens.length > 1) {
+                commandArgs = commandTokens[1].split("/at");
+                commandParsed = new String[commandArgs.length + 1];
+                commandParsed[0] = commandTokens[0];
+                for (int i = 0; i < commandArgs.length; i++) {
+                    commandParsed[i + 1] = commandArgs[i];
+                }
+            }
+            break;
+        default:
+            /**
+             * For commands that do not require further tokenization
+             * e.g. "todo", "list", "bye"
+             */
+            break;
+        }
+        /* Remove any trailing whitespace from the tokens */
+        for (int i = 0; i < commandParsed.length; i++) {
+            commandParsed[i] = commandParsed[i].trim();
+        }
+        return commandParsed;
+    }
+
+    private static String getNumTasksString(int numberOfTasks) {
+        if (numberOfTasks == 1) return "\t You have " + Integer.toString(numberOfTasks) + " task in the list.";
```

Comment:
I believe conditional statements should be placed on a separate line and wrapped with curly brackets even if it is just a line of code. :)